lib/src/components/sidebar/SidebarHeader.js

```diff
@@ -1,10 +1,13 @@
 import * as React from 'react';
-import { Layout } from 'lib';
+import { ButtonIcon, Layout } from 'lib';
 
-export function SidebarHeader({ children, className = '', ...rest }) {
+export function SidebarHeader({ children, className = '', onClose, ...rest }) {
   return (
     <Layout.Header className={`sidebar-header ${className}`} {...rest}>
       <div className="text-overflow-ellipsis">{children}</div>
+      {onClose && (
+        <ButtonIcon name="cross" onClick={onClose} size={ButtonIcon.sizes.md} />
```

Comment:
💚 I can't remember if ButtonIcon allows this, but should we give the icon a title of "Close sidebar" - and maybe allow the passing of a prop to change the title, so we could add different titles from the app, eg. "Close file sidebar"?

Reply:
good shout